src/App.tsx

```diff
@@ -1,61 +1,188 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './Types/user';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import { Todo } from './Types/todo';
+import { FormFields } from './Types/formFields';
+
+const formFields: FormFields = {
+  title: '',
+  user: '0',
+};
+
+const findUserByUserId = (userId: number): User | null => {
+  return usersFromServer.find((user: User) => user.id === userId) || null;
+};
+
+const makeNewTodo = (
+  id: number,
+  title: string,
+  userId: number,
+  user: User | null,
+  completed: boolean = false,
+) => {
+  return {
+    id: id,
+    title: title,
+    completed: completed,
+    userId: userId,
+    user: user,
+  };
+};
+
+const preparedTodos = todosFromServer.map(todo => {
```

Comment:
add type for or variable
```suggestion
const preparedTodos: Todo[] = todosFromServer.map(todo => {
```